src/components/layer-controller/index.js

```diff
@@ -1,2 +1,3 @@
 export { default as LayerControllerSubscriber } from './LayerControllerSubscriber';
-export { default as LayerController } from './LayerController';
+export { default as RasterLayerController } from './RasterLayerController';
+export { default as NonRasterLayerController } from './NonRasterLayerController';
```

Comment:
Would `VectorLayerController` be a good name? Would like to avoid negative names.

Reply:
Good point.